package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "main": "main.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check": "eslint . && prettier -- check ."
```

```suggestion
    "check": "eslint . && prettier --check ."
```

`--check`가 하나의 옵션입니다 `-- check`처럼 띄어쓰기하면 안돼요